### ISSUE TITLE
Motor::operator= and MotorGroup::operator= should not be deleted

### ISSUE THREAD

**Issue by Octogonapus** [OkapiLib/OkapiLib#197]
**Describe the bug**
`Motor::operator=` and `MotorGroup::operator=` (perhaps not the latter?) are deleted because `pros::Motor::operator=` is defined to be something else. We need a version that accepts a `Motor` or `MotorGroup` for copy-assignment purposes.

**Steps To Reproduce**
```
okapi::Motor m1;

void initMotors() {
    m1 = okapi::Motor(1);
}
```

**Comment by Octogonapus** [comment]
This will not be necessary when global ctors are fixed, possibly in the next release.

**Comment by Octogonapus** [comment]
Fixed in kernel v3.1.1.
